config/internal/apiserver/sample-pipeline/sample-pipeline.yaml.tmpl

```diff
@@ -97,21 +88,21 @@ data:
                 \    col_names = [\n        'Sepal_Length', 'Sepal_Width', 'Petal_Length',\
                 \ 'Petal_Width', 'Labels'\n    ]\n    df = pd.read_csv(csv_url, names=col_names)\n\
                 \n    with open(iris_dataset.path, 'w') as f:\n        df.to_csv(f)\n\n"
-              image: quay.io/hukhan/iris-base:1
+              image: quay.io/opendatahub/ds-pipelines-sample-base:v1.0
```

Comment:
we should just use python base here

the docker rate limiting is a pain, but I also don't want to mislead people into thinking they need to use some special base image ... so feel free to point to a quay python if such a thing exists

Reply:
for tracability i added a GH issue here: https://github.com/opendatahub-io/data-science-pipelines-operator/issues/612